src/ext/oblivc/obliv_bits.c

```diff
@@ -26,6 +26,15 @@ static __thread ProtocolDesc *currentProto;
 static inline bool known(const OblivBit* o) { return !o->unknown; }
 
 // --------------------------- Transports -----------------------------------
+
+static bool transportProfilingEnabled = false;
+
+void transportEnableProfiling(bool newVal) {
```

Comment:
"EnableProfiling" sounds like a verb phrase, which shouldn't take in a bool. Either call it "SetProfiling", or have two separate enable/disable functions with no parameter.